requirements/edx/base.txt

```diff
@@ -90,10 +92,11 @@ django-memcached-hashring==0.1.2
 python-openid==2.2.5
 python-dateutil==2.1
 
-# This module gets monkey-patched in third_party_auth.py to fix a Django 1.8 incompatibility.
-# When this dependency gets upgraded, the monkey patch should be removed, if possible.
-# We can also remove the fix to auth_url in third_party_auth/saml.py when that fix is included upstream.
-python-social-auth==0.2.12
+# we have to install social-auth-app-django from fork to get over missing migration
+# issue https://github.com/python-social-auth/social-app-django/issues/28
+# once migrations are run we can switch back to base package(social-auth-app-django==1.0.1)
+-e git+https://github.com/ziafazal/social-app-django.git@8f4688076eab6d9b6fb71bb045e779b6c627835f#egg=social-auth-app-django
```

Comment:
From looking at https://github.com/ziafazal/social-app-django/commit/8f4688076eab6d9b6fb71bb045e779b6c627835f I'm wondering - does your fix work correctly when installing from scratch (with neither 'default' nor 'social_auth' in the history)?

What was the exact error message you were getting? I'm curious why sometime the fix is to remove the `default` migration from the `replaces` and other times it is to remove the `social_auth` migration...

If we have to go through a manual migration process, why not use the upstream recommendation: Just bump to python-social-auth 0.2.21, wait a couple weeks, then bump to the new social_django package?

Also note that in either case, this will eventually cause an error for Open edX and solutions in the upgrade from Eucalyptus to Gingko - manual fix or upgrade script will be required.